context/app/static/js/components/App.jsx

```diff
@@ -7,6 +9,8 @@ import { Header } from './Header';
 
 function App(props) {
   const { flaskData } = props;
+  ReactGA.initialize('UA-133341631-3');
```

Comment:
Did we get the access we want? In the future, might want to pull this into app.conf, but probably not right now.

Reply:
I agree with that but it is indeed not urgent. We did get the access that we asked for.